mobile/src/bundles/users/components/profile-form/profile-form.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+
+import {
+    Button,
+    FlatList,
+    FormField,
+    Input,
+    Selector,
+    View,
+} from '~/bundles/common/components/components';
+import { useAppForm, useCallback } from '~/bundles/common/hooks/hooks';
+import { globalStyles } from '~/bundles/common/styles/styles';
+
+type Properties = {
+    onSubmit: () => void;
+};
+const ProfileForm: React.FC<Properties> = ({ onSubmit }) => {
+    const { control, errors, handleSubmit } = useAppForm({
+        defaultValues: {
+            profileName: '',
+            salaryExpectations: 0,
+            jobTitle: '',
+            currentLocation: '',
+            experience: '',
+        },
+    });
+
+    const jobTitleOptions = [
+        { label: 'Job Title 1', value: 'job1' },
+        { label: 'Job Title 2', value: 'job2' },
+        { label: 'Job Title 3', value: 'job3' },
+    ];
+
+    const currentLocationOptions = [
+        { label: 'Location 1', value: 'location1' },
+        { label: 'Location 2', value: 'location2' },
+        { label: 'Location 3', value: 'location3' },
+    ];
+
+    const handleFormSubmit = useCallback((): void => {
+        void handleSubmit(onSubmit)();
+    }, [handleSubmit, onSubmit]);
+
+    return (
+        <FlatList
```

Comment:
Let's use ScrollView since we know that it's not an endless or long list

Reply:
Selector component uses FlatList and there is a problem with nested scrolling in ScrollView.
Because of this, I switched to FlatList too.

Reply:
Let's try to fix the selector somehow

Reply:
Then I suggest the following implementation of the Selector component: [commit](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/pull/119/commits/744f7a3593621c6c249d2640f3d688b2f7afde1f)